lib/internal/bootstrap/loaders.js

```diff
@@ -285,7 +285,18 @@ NativeModule.prototype.proxifyExports = function() {
     }
   };
 
-  this.exports = new Proxy(this.exports, handler);
+  if (this.exports === process) {
+    if (global.process === process)
+      global.process = this.exports = new Proxy(this.exports, handler);
```

Comment:
I am not sure about this...why is it necessary that `global.process` needs to the same as the one in `import process from 'process'`? In that sense this is going to be inconsistent with `console`?

Also can you add a test if that's the intended behavior?

Reply:
Currently `require('process') === process`. To make these diverge seems like it would only lead to very odd bugs. I think having equivalence is important.